image-layout.md

```diff
@@ -15,9 +15,12 @@ The image layout has two top level directories:
 - "refs" contains [descriptors][descriptors]. Commonly pointing to an [image manifest](manifest.md#image-manifest) or an [image manifest list](manifest-list.md#oci-image-manifest-list-specification).
 
 
-It also contains a file that is used to identify the layout version:
+The image layout MUST contain a file "oci-layout":
 
-- "oci-layout" MUST contain a JSON object with a version field `{"imageLayoutVersion": "1.0.0"}` and MAY include additional fields.
+- It MUST be a JSON object
+- It MUST contain a `imageLayoutVersion` field for determing the version of the layout used
+    - The `imageLayoutVersion` value will align with the `specs.Version` at the time changes to the layout are made, and will pin a given version until changes to the layout are required
```

Comment:
Also, I'd rather change “`specs.Version`” → “OCI Image Format Specification version” to avoid referencing the informative Go from Markdown spec.